pkg/providers/instancetype/instancetype.go

```diff
@@ -290,6 +311,30 @@ func processAvailableResources(az *ecsclient.DescribeAvailableResourceResponseBo
 	}
 }
 
+func getAllVSwitches(client *vpcclient.Client) ([]*vpcclient.DescribeVSwitchesResponseBodyVSwitchesVSwitch, error) {
+	var vSwitchInfo []*vpcclient.DescribeVSwitchesResponseBodyVSwitchesVSwitch
+	runtime := &util.RuntimeOptions{}
+	describeVSwitchesRequest := &vpcclient.DescribeVSwitchesRequest{
+		PageSize: tea.Int32(50),
+	}
```

Comment:
Is the `50` the maximum allowed page size of Aliyun API? Would you mind adding comments mentioning the Aliyun API Docs reference?

Reply:
Done